builder/docker/docker.go

```diff
@@ -0,0 +1,337 @@
+package docker
+
+import (
+	"archive/tar"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Masterminds/cookoo"
+	"github.com/Masterminds/cookoo/log"
+	"github.com/Masterminds/cookoo/safely"
+	"github.com/deis/deis/builder/etcd"
+	docli "github.com/fsouza/go-dockerclient"
+)
+
+// Path to the Docker unix socket.
+var DockSock = "/var/run/docker.sock"
```

Comment:
While not necessary to refactor, in the future docker has this constant defined at https://github.com/docker/docker/blob/51249a3aa09b8e1ade07ed6be0029a190b978b05/opts/opts.go#L27 so we don't have to re-declare this variable :)